helpers/downloader_test.go

```diff
@@ -14,26 +14,26 @@ import (
 
 var _ = Describe("DownloadFile", func() {
 	var url string
+	var sourceDirectory, targetDirectory string
+	var err error
 
 	BeforeEach(func() {
-		directory, err := ioutil.TempDir("", "carrier-test")
-		defer os.Remove(directory)
```

Comment:
The only reason the test was green was because `Remove` refused to delete the dir because it had a file in it :D !
Good catch!